pkg/operator/controllers/routefix/routefix_controller.go

```diff
@@ -45,7 +45,7 @@ type Reconciler struct {
 
 var (
 	verFixed47, _ = version.ParseVersion("4.7.18")  // fixed in 4.7.18
```

Comment:
Unit tests are failing because there is an extra space before the comment causing it to change.  

```suggestion
 	verFixed47, _ = version.ParseVersion("4.7.18") // fixed in 4.7.18 
```

Reply:
:+1: I changed this bit via github web UI. My editor normally reformats things on save. Updated.

Reply:
You know what? I'm actually removing these comments. They are not useful at all.